lib/galaxy/config/schemas/config_schema.yml

```diff
@@ -493,6 +493,42 @@ mapping:
           changes are found, modified tours are automatically reloaded. Takes the same values as the
           'watch_tools' option.
 
+      short_term_storage_dir:
+        type: str
+        default: short_term_web_storage
+        path_resolves_to: cache_dir
+        required: false
+        desc: |
+          Location of files available for a short time as downloads (short term storage).
+          This directory is exclusively used for serving dynamically generated downloadable
+          content. Galaxy may uses the new_file_path parameter as a general temporary directory
+          and that directory should be monitored by a tool such as tmpwatch in production
+          environments. short_term_storage_dir on the other hand is monitored by Galaxy's task
+          framework and should not require such external tooling.
+
+      short_term_storage_default_duration:
+        type: int
+        required: false
+        default: 86400
+        desc: |
+          Default duration before short term web storage files will be cleaned up by Galaxy
+          tasks (in seconds). The default duration is 1 day.
+
+      short_term_storage_maximum_duration:
+        type: int
+        required: false
+        desc: |
+          The maximum duration short term storage files can hosted before they will be marked for
+          clean up. By default there is no default.
```

Comment:
Not having a maximum is breaking `make config-rebuild`:

Traceback (most recent call last):
  File "lib/galaxy/config/config_manage.py", line 558, in <module>
    main()
  File "lib/galaxy/config/config_manage.py", line 244, in main
    action_func(args, app_desc)
  File "lib/galaxy/config/config_manage.py", line 454, in _build_sample_yaml
    _write_sample_section(args, f, app_desc.app_name, schema)
  File "lib/galaxy/config/config_manage.py", line 490, in _write_sample_section
    _write_option(args, f, key, option_value, as_comment=as_comment)
  File "lib/galaxy/config/config_manage.py", line 504, in _write_option
    option, value = _parse_option_value(option_value)
  File "lib/galaxy/config/config_manage.py", line 527, in _parse_option_value
    raise Exception(f"Failed to parse value for {option}, expected int got None")
Exception: Failed to parse value for {'type': 'int', 'unknown_option': False, 'default': None, 'desc': 'The maximum duration short term storage files can hosted before they will be marked for\nclean up. By default there is no default.\n', 'required': False}, expected int got None
make: *** [Makefile:101: config-rebuild] Error 1

Reply:
I missed this comment - I am sorry @nsoranzo